stories/topics/proc-saas-set-up-public.adoc

```diff
@@ -0,0 +1,30 @@
+[id="proc-saas-set-up-public"]
+
+= Subscribing to {SaaSonAWS} from the AWS Marketplace
+
+As an {AWS} (AWS) customer you can search for and subscribe to {SaaSonAWS} from the AWS Marketplace. 
+
+
+.Procedure
+. Log in to your {AWS} account.
+. From the AWS Marketplace search bar, search for "Red Hat subscriptions." 
+** You can select *Red{nbsp}Hat* or *Red{nbsp}Hat Limited*
++
+image::AWS_MP.png[AWS Marketplace search page]
+. From the *Contracts options* page enter your required quantity in the *RHAAP* field.
++
+image::AWS_MP2.png[AWS contracts page]
+Click btn:[View purchase options]-> btn:[Pay now] +
```

Comment:
Don't have these on the same line.
. Click btn:[View purchase options]
. Click btn:[Pay now]